src/api/app/models/package.rb

```diff
@@ -349,6 +349,28 @@ def kiwi_file_md5
     nil
   end
 
+  def changes_files
+    result = []
+    dir_hash.elements('entry') do |e|
+      result << e['name'] if e['name'] =~ /.changes$/
+    end
+    result
+  end
```

Comment:
And what happens with this PR in the meantime?
Actually I don't see why these three lines are a big problem ...
And even to check if this is a multibuild file we need to parse the directory for a changes file. That is even more overhead because I need to parse the directory twice: first to check if it is a multibuild and then to get the changes files. And as said, a normal package can also have several changes files.
So I wouldn't put too much energy in it ...